src/context/font-context.tsx

```diff
@@ -35,7 +35,7 @@ export const FontProvider: React.FC<{ children: React.ReactNode }> = ({
     _setFont(font)
   }
 
-  return <FontContext value={{ font, setFont }}>{children}</FontContext>
+  return <FontContext.Provider value={{ font, setFont }}>{children}</FontContext.Provider>
```

Comment:
But this is a valid usage starting from [React 19](https://react.dev/blog/2024/12/05/react-19#context-as-a-provider).

```tsx
<FontContext value={{ font, setFont }}>{children}</FontContext>
```

Plus, `<FontContext.Provider>`  will be deprecated in the future.

Reply:
Because of this, I'll be closing this PR.
Thanks.